app/src/main/java/org/gdg/frisbee/android/app/GoogleApiClientFactory.java

```diff
@@ -26,12 +26,7 @@ private static GoogleApiClient createClient(Context context, boolean withSignIn)
             .addApi(AppIndex.API);
 
         if (withSignIn) {
-            Games.GamesOptions gamesOptions = Games.GamesOptions.builder()
-                .setRequireGooglePlus(true)
-                .setShowConnectingPopup(false).build();
-
             builder.addApi(Plus.API).addScope(Plus.SCOPE_PLUS_PROFILE)
-                .addApi(Games.API, gamesOptions)
                 .addApi(Drive.API).addScope(Drive.SCOPE_APPFOLDER);
```

Comment:
We are using this to save game state. I think we can remove it as well. If we want to store something we can use Firebase.